src/executables/util_cs.c

```diff
@@ -4649,9 +4652,71 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  /* execute phase */
+  error_code = mmon_get_server_info (server_info);
+  if (error_code != NO_ERROR)
+    {
+      goto error_exit;
+    }
+
+  // TODO: mmon_print_server_info (server_info);
+
+  if (module_index >= 0)
+    {
+      error_code = mmon_get_module_info (module_index, module_info);
+      if (error_code != NO_ERROR)
+	{
+	  goto error_exit;
+	}
+
+      // TODO: mmon_print_module_info (module_info);
+    }
+  else
+    {
+      if (print_show_all)
+	{
+	  module_count = MMON_MODULE_LAST;
+	  print_transaction = true;
```

Comment:
The `print_transaction` is always true if `module_index < 0`. We can take it out of this branch.

Reply:
Done.